### DISCUSSION TITLE
MySports German channels

### DISCUSSION THREAD

**Discussion by RKaefer** [iptv-org/community#488] [Q&A]
Hey, i'm kinda new to IPTV in general.

Anyway, in the Database channels.csv, i saw the different MySports channels listed, but when i looked for them in the indes.m3u file or sports category m3u file i coudn't find the channels. Why is that so, does the channel.csv file simply list a whole lot of TV channels even if no source is currently know for it?

Anyway thanks for the help.

**Comment by freearhey** [comment]
> Why is the channel on the iptv-org.github.io but not in the playlist?
>
> The site contains a list of all TV channels in the world and only those of them for which we have working stream links are included in the playlists.

https://github.com/iptv-org/iptv/blob/master/FAQ.md#why-is-the-channel-on-the-iptv-orggithubio-but-not-in-the-playlist
